src/aics-image-viewer/components/App/index.tsx

```diff
@@ -143,435 +134,227 @@ const defaultProps: AppProps = {
-    let lutObject = histogram.lutGenerator_percentiles(LUT_MIN_PERCENTILE, LUT_MAX_PERCENTILE);
-    // and if init settings dictate, recompute it:
-    if (initViewerSettings) {
-      const initSettings = findFirstChannelMatch(name, channelIndex, initViewerSettings);
-      if (initSettings) {
-        if (initSettings.lut !== undefined && initSettings.lut.length === 2) {
-          let lutmod = "";
-          let lvalue = 0;
-          let lutvalues = [0, 0];
-          for (let i = 0; i < 2; ++i) {
-            const lstr = initSettings.lut[i];
-            // look at first char of string.
-            let firstchar = lstr.charAt(0);
-            if (firstchar === "m" || firstchar === "p") {
-              lutmod = firstchar;
-              lvalue = parseFloat(lstr.substring(1)) / 100.0;
-            } else {
-              lutmod = "";
-              lvalue = parseFloat(lstr);
-            }
-            if (lutmod === "m") {
-              lutvalues[i] = histogram.maxBin * lvalue;
-            } else if (lutmod === "p") {
-              lutvalues[i] = histogram.findBinOfPercentile(lvalue);
-            }
-          }
+  const getOneChannelSetting = (channelName: string, settings?: ChannelState[]): ChannelState | undefined => {
+    return (settings || getChannelSettings()).find((channel) => channel.name === channelName);
+  };
 
-          lutObject = histogram.lutGenerator_minMax(
-            Math.min(lutvalues[0], lutvalues[1]),
-            Math.max(lutvalues[0], lutvalues[1])
-          );
-        }
-      }
-    }
+  const setOneChannelLoaded = (index: number): void => {
+    const newLoadedChannels = getLoadedChannels().slice();
+    newLoadedChannels[index] = true;
+    setLoadedChannels(newLoadedChannels);
+  };
 
-    const newControlPoints = lutObject.controlPoints.map((controlPoint) => ({
-      ...controlPoint,
-      color: TFEDITOR_DEFAULT_COLOR,
-    }));
-    aimg.setLut(channelIndex, lutObject.lut);
-    return newControlPoints;
-  }
+  // Image loading/initialization functions ///////////////////////////////////
 
-  onChannelDataLoaded(
+  const onChannelDataLoaded = (
     aimg: Volume,
     thisChannelsSettings: ChannelState,
     channelIndex: number,
-    keepLuts: boolean | undefined
-  ): void {
-    const { image, view3d } = this.state;
-    if (!view3d || aimg !== image) {
-      return;
-    }
-    const volenabled = thisChannelsSettings.volumeEnabled;
-    const isoenabled = thisChannelsSettings.isosurfaceEnabled;
-    view3d.setVolumeChannelOptions(aimg, channelIndex, {
-      enabled: volenabled,
-      color: thisChannelsSettings.color,
-      isosurfaceEnabled: isoenabled,
-      isovalue: thisChannelsSettings.isovalue,
-      isosurfaceOpacity: thisChannelsSettings.opacity,
-    });
-
+    keepLuts = false
+  ): void => {
     // if we want to keep the current control points
+    // TODO this function is never called with `keepLuts = true`. Should it ever be? On FOV switch e.g.?
```

Comment:
there are definitely scenarios where the viewer should be told to load fresh volume data but keep the per-channel settings the same (for example, playing through a time lapse).  However, I don't know how `keepLuts` would be populated in that scenario yet.  

There is probably some version of this codebase's history where keepLuts was set to true - my guess is it was associated with the "nextImg/prevImg" workflow.